test/views/CashAccountTransactionsNotAvailableSpec.scala

```diff
@@ -132,6 +136,19 @@ class CashAccountTransactionsNotAvailableSpec extends ViewTestHelper {
   }
 
   private def shouldContainPaymentSection(implicit view: Document): Assertion =
-    view.text().contains(messages("cf.cash-account.detail.transactions-not-available")) mustBe true
+    view.text().contains(messages("cf.cash-account.detail.transactions-not-available")) mustBe false
+
+  private def shouldContainHelpAndSupportSection(implicit view: Document): Assertion =
```

Comment:
Please format from line 142-153 as these are not indented correctly. Also the assertions inside shouldContainHelpAndSupportLink should be moved inside shouldContainHelpAndSupportSection as those are all part of shouldContainHelpAndSupportSection. 

shouldContainHelpAndSupportSection should also be included as part of test named "transactionsTimeout is true"